src/components/modals/EntityCreatorEditor/EntityCreatorContainer.tsx

```diff
@@ -502,6 +597,23 @@ class Container extends React.Component<Props, ComponentState> {
         })
     }
 
+    @OF.autobind
+    onCancelEnumDelete() {
+        this.setState({
+            deleteEnumCheck: null
+        })
+    }
+
+    @OF.autobind
+    onConfirmEnumDelete() {
+        if (this.state.deleteEnumCheck) {
```

Comment:
Why do we need the `if` statement?

I thought the design was such that this window to confirm the deletion of an enum in use should only be open when there is an enum assigned to.  

If this is true, you could should never be able to click confirm when `deleteEnumCheck` is null.  
If this is false and there is a case where they can be out of sync, the user would not be able to close the window. Since the if statement would skip the logic in here and clicking confirm would not do anything.

Reply:
Fixed so can still close the window